packages/zoe/src/zoeService/zoe.js

```diff
@@ -60,6 +62,15 @@ const makeZoeKit = (
     startInstanceFee: 0n,
     offerFee: 0n,
   },
+  meteringConfig = {
+    incrementBy: 25_000_000n,
+    initial: 50_000_000n, // executeContract for treasury is 13.5M
+    threshold: 25_000_000n,
+    price: {
+      feeNumerator: 1n,
+      computronDenominator: 1n, // default is just one-to-one
+    },
```

Comment:
RUN is a stablecoin pegged to the dollar? What units are RUN in? With these settings, what would the price be of, for example, treasury's 13.5M computrons?

I know we don't need realistic prices at this point. But could we have prices that were within an order of magnitude or two of what we think may be realistic?

Reply:
A unit of RUN displayed to the user is meant to be $1. Because the decimalPlaces are 6, that would be 1_000_000 actual RUN units. So with one-to-one feeNumerator to computronDenominator, that would come to about $12.90 by my calculations if it takes about 12.9M computrons. Maybe this should be $1.29 instead, but with these settings we're within an order of magnitude or two.